src/core/LoadingSpinner/LoadingSpinner.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { Component, forwardRef } from 'react';
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import { baseStyles } from './LoadingSpinner.baseStyles';
+import { SuomifiThemeProp, SuomifiThemeConsumer } from '../theme';
+import { HtmlDiv, HtmlDivWithRef } from '../../reset';
+import { VisuallyHidden } from '../VisuallyHidden/VisuallyHidden';
+import { ComponentIcon } from '../StaticIcon/StaticIcon';
+import { Icon } from '../../core/Icon/Icon';
+
+export type LoadingSpinnerStatus = 'loading' | 'success' | 'failed';
+export interface LoadingSpinnerProps {
+  /** Custom class name for styling and customizing */
+  className?: string;
+  /** Unique id */
+  id?: string;
+  /** Status text */
+  text: string;
+  /** Align text on bottom or on the right side of animation svg
+   * @default 'bottom'
+   */
+  textAlign?: 'bottom' | 'right';
+  /** Is the text visible or hidden
```

Comment:
This wording differs from the descriptions in rest of the library somewhat. Maybe "Show or hide status text" or something along those lines could work better.

Reply:
Fixed